src/includes/getting-started-config/javascript.react.mdx

```diff
@@ -0,0 +1,121 @@
+You should initialize the Sentry React SDK as soon as possible during your application load up, before initializing React:
```

Comment:
delete this line (it's covered by the code sample and the preceding prose from the common content)